tools/InstallerSpec/InstallerSpec/Program.cs

```diff
@@ -213,22 +216,35 @@ static void Main(string[] args)
                             {
                                 while (!streamReader.EndOfStream)
                                 {
+
                                     // WARNING: The following codes are written with the assumption that 
                                     // files with listed names are residing under "binpath", but this may 
                                     // not be true in the future. Rewrite this logic and make "textFilePath"
                                     // contains fully qualified paths instead of doing the prefix here.
                                     // 
                                     var fileNameWithoutPath = streamReader.ReadLine().Trim();
+                                    //this is a comment, ignore this line
+                                    if (fileNameWithoutPath.StartsWith("##") || string.IsNullOrWhiteSpace(fileNameWithoutPath))
+                                    {
+                                        continue;
+                                    }
                                     var fileNameWithPath = Path.Combine(binpath, fileNameWithoutPath);
                                     if (!string.IsNullOrEmpty(fileNameWithPath))
```

Comment:
Might be too much to ask.. Can we file a follow up task to include folder pattern? I would like to add .\libG* or something like that for future bullet proof.

Reply:
I looked into that, .net does not have any glob pattern support - they instead have a nuget package you can reference: 
https://www.nuget.org/packages/Microsoft.Extensions.FileSystemGlobbing/

I didn't want to add additional deps for this now.

Reply:
@mjkkirschner Yes, I think this PR is good in the current state. Let's do the improvement later.